src/common/OutputParsersUtils.cpp

```diff
@@ -0,0 +1,28 @@
+
+#include "OutputParsersUtils.h"
+
+namespace ParsersUtils
+{
+
+    namespace
+    {
+        QStringList cleanup(const QStringList& list)
```

Comment:
Is it necessary using QT types, not STD ones?

Reply:
Qt's containers have more useful stuff + this function will cooperate with qt code